angular.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "$schema": "../node_modules/@angular/cli/lib/config/schema.json",
```

Comment:
Required an additional dot to indicate that the actual file is a folder farther up, good work.